isamples_metadata/GEOMETransformer.py

```diff
@@ -4,12 +4,34 @@
     Transformer,
 )
 
+TISSUE_ENTITY = "Tissue"
+
+
+# Forward declarations appear to be the cleanest way to manage the circular dependency between these two classes
+class GEOMETransformer(Transformer):
+    pass
+
+
+class GEOMEChildTransformer(GEOMETransformer):
+    pass
+
 
 class GEOMETransformer(Transformer):
     """Concrete transformer class for going from a GEOME record to an iSamples record"""
 
+    def __init__(self, source_record: typing.Dict):
+        super().__init__(source_record)
+        self._child_transformers = []
+        children = self._get_children()
+        for child_record in children:
+            entity = child_record.get("entity")
+            if entity == TISSUE_ENTITY:
+                self._child_transformers.append(
+                    GEOMEChildTransformer(source_record, child_record)
+                )
+
     def transform(self) -> typing.Dict:
-        transformed_record = super(GEOMETransformer, self).transform()
+        transformed_record = super().transform()
```

Comment:
In reading about how `super` works in Python, this is the preferred way to do it in newer Python versions, so this is just cleanup.

Reply:
Yes, correct.